machine/v1beta1/types_awsprovider.go

```diff
@@ -355,3 +371,22 @@ const (
 	// When set to CapacityBlock the instance utilizes pre-purchased compute capacity (capacity blocks) with AWS Capacity Reservations.
 	MarketTypeCapacityBlock MarketType = "CapacityBlock"
 )
+
+// HostAffinity describes the host affinity of an EC2 Instance
+// +kubebuilder:validation:Enum:=host;default;""
+// +kubebuilder:default=""
```

Comment:
How is setting `""` different than omitting the field that uses this type altogether?